velox/functions/prestosql/tests/StringFunctionsTest.cpp

```diff
@@ -1361,6 +1361,37 @@ TEST_F(StringFunctionsTest, HmacSha512) {
   EXPECT_EQ(std::nullopt, hmacSha512(std::nullopt, "velox"));
 }
 
+TEST_F(StringFunctionsTest, HmacMd5) {
```

Comment:
first alphabet of test name should not be capital. Please use camel Case name hmacMd5.

Reply:
All other function names in this file use the same PascalCase.

Reply:
You are right the Hmac test functions violate this. The convention is camelCase like spookyHash... tests above.

Also please take a look at the coding guidelines at https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md for more information.